### DISCUSSION TITLE
Calcualtion of Enthalpy  Ambient in the source file

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#12998] [Q&A]
Dear developers,

I viewed the material properties against the `.out` file. In my example, given the specific heat, density and thermal conductivity as constant as follows.
```fortran
&MATL ID            = 'STEEL'
      DENSITY       = 7850.
      CONDUCTIVITY  = 45.8
      SPECIFIC_HEAT = 0.46 /
```
I viewed the Enthalpy (kJ/kg) Ambient from the `.out` file and was curious how it was calculated. Therefore, I checked the source code, and the calculation of enthalpy (`ML%H`) is only found in `read.f90`, but its calculation puzzles me as to how it is possible to directly add enthalpy to specific heat. 
From the LINE 6973 to LINE 6981, they show,
```fortran
   ! Check units of heat of reaction

   ALLOCATE(ML%H(0:I_MAX_TEMP)) ; ML%H(0) = 0._EB
   DO I=1,I_MAX_TEMP
      ML%H(I) = ML%H(I-1) + ML%C_S(I)
   ENDDO
   CALL INTERPOLATE1D_UNIFORM(0,ML%H,ML%REFERENCE_ENTHALPY_TEMPERATURE,ANS)
   H_ADJUST = ML%REFERENCE_ENTHALPY - ANS
   ML%H = ML%H + H_ADJUST
```
I calculate it by multiplying the specific heat by the temperature and can get the same value in the `.out` file. 
![image](https://github.com/firemodels/fds/assets/96673442/466070ad-b47c-4fea-b1ce-9357e561cb9a)

I would appreciate it if you can give some guidance on this.

Kind regards,

LI

**Comment by drjfloyd** [comment]
Read the Appendix on material enthalpy in the Technical Reference.
